src/Type/CoreConfigProvider.php

```diff
@@ -137,7 +137,7 @@ public function render(
             'driver' => [
                 'orm_default' => [
                     'drivers' => [
-                        '{$this->fileSystem->getModuleName()}\\{$entity->getClassName()}\\Entity' => '{$entity->getClassName()}Entities',
+                        'Core\\{$entity->getClassName()}\\Entity' => '{$entity->getClassName()}Entities',
```

Comment:
```suggestion
                        '{$entity->getNamespace()}' => '{$entity->getClassName()}Entities',
```
You can use the namespace for this.
Other than that, good catch! 👍